docs/examples/brep_explorer.py

```diff
@@ -34,8 +34,8 @@
     viewer.add(edge.to_line(), linewidth=5, linecolor=Color(0.2, 0.2, 0.2))
 
 for face in faces:
-    brep = BRep()
-    brep.occ_shape = face.occ_face
+    brep = Brep()
+    brep.native_brep = face.occ_face
```

Comment:
we should change the Brep API to always return Breps and add a special method to get a specific subtype. for example, `Brep.as_face()`, or something like that.